tests/unit_tests/test_wave_06.py

```diff
@@ -261,11 +253,37 @@ def test_swap_best_by_category_no_other_match_is_false():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is falsy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories
+    assert not result
```

Comment:
A check to see if `result` is falsey doesn't actually test that `result` is what we expect it to be. How could you refactor this assertion so that you're testing for the accurate scenario as outlined by the README?